Running-Mastodon/Development-guide.md

```diff
@@ -128,6 +128,23 @@ Go to http://localhost:3000 to see your development instance.
 
 Admin account is `admin@localhost:3000`. Password is `mastodonadmin`.
 
+## Bash on Windows
+
+To run Mastodon on [Bash on Ubuntu on Windows](https://msdn.microsoft.com/en-us/commandline/wsl/about), you can follow the Ubuntu instructions with some small tweaks.
```

Comment:
yeah totally, I meant even for development. Like, that we don't support users who have trouble with development on bash on windows.